.github/workflows/homebrew.yml

```
@@ -55,18 +48,25 @@ jobs:
           else
-          echo "asd.rb not found"
+            echo "asd.rb not found"
           fi
 
       - name: Commit and push if it changed
         run: |
           git config --local user.email "action@github.com"
           git config --local user.name "GitHub Action"
-          git commit -am "Update to ${{ steps.latest_release.outputs.tag }}" --allow-empty
-          git push
+        run: |
```

actionlint (1.7.4)

key "run" is duplicated in element of "steps" section. previously defined at line:59,col:9

(syntax-check)